packages/oscal-react-library/src/components/OSCALBackMatter.tsx

```diff
@@ -0,0 +1,178 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Chip from "@mui/material/Chip";
+import Grid from "@mui/material/Grid";
+import { Typography } from "@mui/material";
+import FormatQuoteIcon from "@mui/icons-material/FormatQuote";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+import StyledTooltip from "./OSCALStyledTooltip";
+import { getAbsoluteUrl, guessExtensionFromHref } from "./oscal-utils/OSCALLinkUtils";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
+import { OSCALMarkupLine } from "./OSCALMarkupProse";
+import OSCALAnchorLinkHeader from "./OSCALAnchorLinkHeader";
+import OSCALEditableTextField from "./OSCALEditableTextField";
+import { Resource, ResourceLink, BackMatter } from "@easydynamics/oscal-types";
+import { ReactElement } from "react";
+
+export const OSCALBackMatterCard = styled(Card)(
+  ({ theme }) => `
+    margin-top: ${theme.spacing(1)};
+    display: flex;
+    flex-direction: column;
+`
+);
+
+function TitleDisplay(props: any): ReactElement {
+  const { children, uuid } = props;
+  return (
+    <OSCALAnchorLinkHeader value={uuid}>
+      <Typography variant="subtitle1">{children}</Typography>
+    </OSCALAnchorLinkHeader>
+  );
+}
+
+function CitationDisplay(props: any): ReactElement {
+  if (!props.resource?.citation?.text) {
+    return (
+      <FormatQuoteIcon
+        color="disabled"
+        fontSize="small"
+        titleAccess={`${props.resource.title}-citation`}
+      />
+    );
+  }
+  return (
+    <StyledTooltip title={<OSCALMarkupLine>{props.resource.citation.text}</OSCALMarkupLine>}>
+      <FormatQuoteIcon
+        color="primary"
+        fontSize="small"
+        titleAccess={`${props.resource.title}-citation`}
+      />
+    </StyledTooltip>
+  );
+}
```

Comment:
This code actually is problematic because if `props.resource` is undefined, we are still trying to access its `title` attribute in the `titleAccess` property of `FormatQuoteIcon`... Can we just make `resource` required like this? If not, we'll have to come up with something else to do with `props.resource` is undefined.

```suggestion
interface CitationDisplayProps {
  readonly resource: Resource;
}

function CitationDisplay(props: CitationDisplayProps): ReactElement {
  if (!props.resource.citation?.text) {
    return (
      <FormatQuoteIcon
        color="disabled"
        fontSize="small"
        titleAccess={`${props.resource.title}-citation`}
      />
    );
  }
  return (
    <StyledTooltip title={<OSCALMarkupLine>{props.resource.citation.text}</OSCALMarkupLine>}>
      <FormatQuoteIcon
        color="primary"
        fontSize="small"
        titleAccess={`${props.resource.title}-citation`}
      />
    </StyledTooltip>
  );
}
```